Services/Services/UserService.cs

```diff
@@ -1,14 +1,22 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Services.DTOs;
+﻿using Services.DTOs;
+using Data.Entities;
+using Data.Repositories;
 
 namespace Services.Services
 {
-    internal class UserService
+    public class UserService
     {
+
+        private IUserRepository _userRepository;
+
+        public UserService(IUserRepository userRepository) { _userRepository = userRepository; }
+        public string GetEmailOfUser(int id)
```

Comment:
Do you need this method?

Reply:
This method will return email and i will be injecting it to Controller to get emails form db.